src/text/raw_text_writer.rs

```diff
@@ -107,12 +117,9 @@ impl RawTextWriterBuilder {
             annotations: Vec::new(),
             field_name: None,
             containers: vec![EncodingLevel::default()],
-            // TODO: We should consider putting these in a single struct (`WhitespaceConfig`?)
-            //       and `Box`ing it. Each `String` is 24 bytes, making the writer much bigger.
-            space_between_values: self.space_between_values,
-            indentation: self.indentation,
-            space_after_field_name: self.space_after_field_name,
-            space_after_container_start: self.space_after_container_start,
+            // Should we validate here that all the strings in `whitespace_config` actually are
+            // semantically whitespace?
```

Comment:
Good question! Let's open an issue for it and link that here as a `TODO`.

Reply:
Done: https://github.com/amzn/ion-rust/issues/438